vonk/features/hipaa.rst

```diff
@@ -0,0 +1,109 @@
+.. _feature_hipaa:
+
+HIPAA compliance
+================
+
+Vonk is well-tested, secure HL7 FHIR® server that enables you to comply with the Technical Safeguards of the HIPAA Security Rule.
```

Comment:
```suggestion
Vonk is the well-tested, secure HL7 FHIR® server that enables you to comply with the Technical Safeguards of the HIPAA Security Rule.
```

Reply:
Oops - actually an "a" is more appropriate here. Thanks for the catch.

Reply:
I started with 'a' and then thought 'what the heck, lets be bold' :-)